java/ql/lib/semmle/code/java/dataflow/internal/DataFlowUtil.qll

```diff
@@ -118,6 +121,16 @@ predicate localFlowStep(Node node1, Node node2) {
  */
 cached
 predicate simpleLocalFlowStep(Node node1, Node node2) {
```

Comment:
Perhaps wrap `localFlowStep` and `simpleLocalFlowStep` in a cached internal module to make sure they are evaluated together.